code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -171,7 +201,7 @@
 	icon_icon = 'icons/mob/actions/actions_animal.dmi'
 	button_icon_state = "riot"
 	background_icon_state = "bg_clock"
-	cooldown_time = 80
+	cooldown_time = 4 SECONDS
```

Comment:
this isn't mentioned in the changelog

Reply:
20 ticks - 1 second, no?

So, 4 seconds.

Reply:
cooldowns are timed against world.time which is in deciseconds. besides that: the 20 ticks you refer to is the speed of the mob subsystem which is 20, but thats also in deciseconds, Life() is called once every 2 seconds, but we use deltatime to make sure that any value used represents a value of every 1 second. (Aka its multiplied by 2).

Basically, change this to 8 SECONDS if the change in balance was unintentional :)

Reply:
![image](https://user-images.githubusercontent.com/29008542/104482670-33a6f580-5595-11eb-882d-dff01bbaddfc.png)
i think someone got confusy woosied ;)

Reply:
> Basically, change this to 8 SECONDS if the change in balance was unintentional :)

Thank you, I didn't know that!

Reply:
I reverted to ticks. It's standardized this way.